### ISSUE TITLE
Where's the expected method getSelecteds() ?!

### ISSUE THREAD

**Issue by caverna** [rstaib/jquery-bootgrid#121]
Ok, I can set a field to be 'selectable' with data-identifier="true", but and what else?
I'm using a dirt tricks to get the ids:
```
<form id='myDirtForm'>
    <table id='myTable'>
        <thread>
            <th data-column-id="id" data-identifier="true" data-visible="false">Id</th>
            <th data-column-id="name">Id</th>
            ...
        </thread>
    </table>
</form>
<script>
    var dataToBeManipuled = $('#myDirtForm').serializeArray();
    ...
</script>
```
Unfortunately this trick doesn't work when I'm using pages.
What's the point to be able to select rows if I can get that selection back?
Looking at the source, I found the variable 'selectedRows', but I couldn't expose it.
Any help?

**Comment by caverna** [comment]
Scratching around I got this:
```
Grid.prototype.getSelected = function (result) {
        for (var i = 0; i < this.selectedRows.length; i++) {
            result.push(this.selectedRows[i]);
        }
        return this;
    }
```
And to invoke, I need to do this:
```
var result=[];
$('#myTable').bootgrid('getSelected', result);
```
It's ugly but gets the job done...
I did try 'return result;' but I only get the complete table entity.

**Comment by rstaib** [comment]
Implemented in version 1.2.0 which will be publicly awailable within the next few days.

**Comment by rstaib** [comment]
I have recently published the release candidate (prerelease version including all features) to NPM and NuGet. Would be nice and very appreciated to get your feedback before publishing it.
